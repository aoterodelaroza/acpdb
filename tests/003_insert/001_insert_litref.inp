## check: 001_insert_litref.out -a1e-10
## delete: 001_insert_litref.db
## labels: regression quick

## connect
system rm -f 001_insert_litref.db
connect 001_insert_litref.db

insert litref example
  authors A. author, O. guy
  title This is the title
  journal J. Journals
  volume 11
  page 223840
  year 1990
  doi example.com/1234
  description An example reference.
end
insert litref example2
  authors x2 A. author, O. guy
  title x2 This is the title
  journal x2 J. Journals
  volume x2 11
  page x2 223840
  year x2 1990
  doi x2 example.com/1234
  description x2 An example reference.
end
insert litref example3
  authors x3 A. author, O. guy
  title x3 This is the title
  journal x3 J. Journals
  volume x3 11
  page x3 223840
  year x3 1990
  doi x3 example.com/1234
  description x3 An example reference.
end
insert litref example4
  authors x4 A. author, O. guy
  title x4 This is the title
  journal x4 J. Journals
  volume x4 11
  page x4 223840
  year x4 1990
  doi x4 example.com/1234
  description x4 An example reference.
end

print

delete litref 1 example2
delete litref

print

