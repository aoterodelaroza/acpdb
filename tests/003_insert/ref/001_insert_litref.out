* SYSTEM: rm -f 001_insert_litref.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 001_insert_litref.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)
# INSERT LITREF example

* INSERT: insert data into the database (LITREF)
# INSERT LITREF example2

* INSERT: insert data into the database (LITREF)
# INSERT LITREF example3

* INSERT: insert data into the database (LITREF)
# INSERT LITREF example4

* PRINT: print the contents of the database 

# Table of property types
| id | key | description |
| 1 | energy_difference | A difference of molecular or crystal energies (reaction energy, binding energy, lattice energy, etc.) |
| 2 | energy | The total energy of a molecule or crystal |

# Number of literature references: 4

# Table of methods
| id| key| gaussian_keyword| psi4_keyword| litrefs| description|

# Table of sets
| id| key| property_type| litrefs| description|

# Number of properties and structures in each set
| id | key | properties | structures |

## Description of the training set
# The TRAINING SET is NOT DEFINED
--- No atoms found (ATOM) ---
--- No angular momenta found (LMAX) ---
--- No exponents found (EXP) ---
--- No subsets found (SUBSET) ---
--- No weights found (W) ---
--- No empty method found (EMPTY) ---
--- No reference method found (REFERENCE) ---

## Table of saved training sets in the database
| Name |

